ambry-api/src/main/java/com/github/ambry/config/StoreConfig.java

```diff
@@ -678,5 +686,7 @@ public StoreConfig(VerifiableProperties verifiableProperties) {
 
     storeDeletedPutAsDeleteInFindEntries =
         verifiableProperties.getBoolean(storeDeletedPutAsDeleteInFindEntriesName, false);
+    storePersistRemoteTokenIntervalInSeconds =
+        verifiableProperties.getIntInRange(storePersistRemoteTokenIntervalInSecondsName, 0, 60 * 1, 60 * 60 * 24);
```

Comment:
this would probably not going to work, the default value is no in the range.

Reply:
Nice catch. Thank you.